README.md

```diff
@@ -95,3 +95,31 @@ The build is minified and the filenames include the hashes.
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run docker:build-image`
+
+Builds the Docker image based on already built application with `npm run build`.
+
+Supported environmental variables:
+
+- `DOCKER_REPOSITORY` (default: `ghcr.io/capactio`)
+- `DOCKER_TAG` (default: `latest`)
+
+### `npm run docker:push-image`
+
+Pushes the Docker image.
+
+Supported environmental variables:
+
+- `DOCKER_REPOSITORY` (default: `ghcr.io/capactio`)
+- `DOCKER_TAG` (default: `latest`)
+
+### Run built Docker image
+
+To run built Docker image, execute:
+
+```bash
+docker run --rm -it -p 3000:80 ghcr.io/capactio/dashboard:latest
+```
+
+The app listens on port `3000`.
```

Comment:
```suggestion
To access Dashboard open [http://127.0.0.1:3000](http://127.0.0.1:3000).
```
or 
```suggestion
Now you can access Dashboard by opening [http://127.0.0.1:3000](http://127.0.0.1:3000).
```
or sth similar what gives us an option to easily open tab with dashboard :)